setup.py

```diff
@@ -34,12 +34,11 @@
     ]
 }
 
-extras_require['dev'] = (
-    extras_require['dev'] +
-    extras_require['test'] +
-    extras_require['lint'] +
-    extras_require['doc']
-)
+
+extras_require['dev'].extend(extras_require['test'])
+extras_require['dev'].extend(extras_require['lint'])
+extras_require['dev'].extend(extras_require['doc'])
+
```

Comment:
Mmmh...this is how we do it in all projects. Are you saying we should change this across the board in all repos? I don't have a strong opinion since it's only in the `setup.py` file but I want to bring up that:

1. This is more repetitive and imho harder to parse
2. If we adopt it, I think it should be adopted everywhere including our template project (paging in @carver for another opinion)

Reply:
We have been experimenting with other options, but definitely not settled on anything. Here is another approach that I think worked okay: https://github.com/ethereum/eth-abi/pull/130#discussion_r279082896

Reply:
Otherwise, I think the linter will be happy with just moving the `+` to the beginning of the line, which maybe seems cleaner than all the extends.

Or yet another option:
```py
extras_require['dev'].extend([
    extras_require['test'],
    extras_require['lint'],
    extras_require['doc'],
])

Reply:
My linter doesn't like the `+` at the beginning or end... :frowning_face:

Reply:
I have things I dislike about all of our approaches.... I'm fine with us normalizing this across all repos but I think we need to have a method we like before we can do that...

Reply:
> My linter doesn't like the `+` at the beginning or end...

Which linter? It's pretty strange to not allow multiline addition...

Reply:
synstastic in vim.  `W503` if it's before.  `W504` if it's after.

Reply:
Yeah, seems like we should just do what flake8 does everywhere, until we decide to make portfolio-wide changes about how `flake8` works. IIRC, flake8 complains about trailing but not leading `+`s.

Reply:
You are probably ignoring `W503` then if yours is not complaining about leading.

Reply:
I don't have any local overrides. I had it backwards though:

```py
a = (
    1 +
    2 +
    3
)
```

Doesn't raise any `flake8` errors, using the `flake8` settings from py-evm, for example.